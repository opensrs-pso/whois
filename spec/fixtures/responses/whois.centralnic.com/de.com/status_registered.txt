This whois service is provided by CentralNic Ltd and only contains
information pertaining to Internet domain names we have registered for
our customers. By using this service you are agreeing (1) not to use any
information presented here for any purpose other than determining
ownership of domain names, (2) not to store or reproduce this data in 
any way, (3) not to use any high-volume, automated, electronic processes
to obtain data from this service. Abuse of this service is monitored and
actions in contravention of these terms will result in being permanently
blacklisted. All data is (c) CentralNic Ltd https://www.centralnic.com/

Domain ID:CNIC-DO819205
Domain Name:BOERSE.DE.COM
Created On:28-Nov-2011 21:38:30 UTC
Last Updated On:07-Jan-2013 13:10:46 UTC
Expiration Date:28-Nov-2012 23:59:59 UTC
Status:PENDING DELETE
Status:PENDING DELETE RESTORABLE
Registrant ID:T2187-GANDI-VKXS
Registrant Name:Dhananjeyan Thangavelu
Registrant Organization:LTV Global Inc.
Registrant Street1:5 Meenakshi Nagar
Registrant City:Trichy
Registrant State/Province:TN
Registrant Postal Code:620006
Registrant Country:IN
Registrant Phone:+91.4312431415
Registrant Email:08cb0ba28467bf1e9ec03c83fdc62534-1376264@contact.gandi.net
Admin ID:T2187-GANDI-VKXS
Admin Name:Dhananjeyan Thangavelu
Admin Organization:LTV Global Inc.
Admin Street1:5 Meenakshi Nagar
Admin City:Trichy
Admin State/Province:TN
Admin Postal Code:620006
Admin Country:IN
Admin Phone:+91.4312431415
Admin Email:08cb0ba28467bf1e9ec03c83fdc62534-1376264@contact.gandi.net
Tech ID:T2187-GANDI-VKXS
Tech Name:Dhananjeyan Thangavelu
Tech Organization:LTV Global Inc.
Tech Street1:5 Meenakshi Nagar
Tech City:Trichy
Tech State/Province:TN
Tech Postal Code:620006
Tech Country:IN
Tech Phone:+91.4312431415
Tech Email:08cb0ba28467bf1e9ec03c83fdc62534-1376264@contact.gandi.net
Billing ID:T2187-GANDI-VKXS
Billing Name:Dhananjeyan Thangavelu
Billing Organization:LTV Global Inc.
Billing Street1:5 Meenakshi Nagar
Billing City:Trichy
Billing State/Province:TN
Billing Postal Code:620006
Billing Country:IN
Billing Phone:+91.4312431415
Billing Email:08cb0ba28467bf1e9ec03c83fdc62534-1376264@contact.gandi.net
Sponsoring Registrar ID:H1167922
Sponsoring Registrar IANA ID:81
Sponsoring Registrar Organization:Gandi SAS
Sponsoring Registrar Street1:63-65 bd. Massena
Sponsoring Registrar City:Paris
Sponsoring Registrar Postal Code:75013
Sponsoring Registrar Country:FR
Sponsoring Registrar Phone:+33 1 7039 3740
Sponsoring Registrar FAX:+33 1 4373 1851
Sponsoring Registrar Website:http://www.gandi.net/
Name Server:B.DNS.GANDI.NET
Name Server:C.DNS.GANDI.NET
Name Server:A.DNS.GANDI.NET
DNSSEC:Unsigned


