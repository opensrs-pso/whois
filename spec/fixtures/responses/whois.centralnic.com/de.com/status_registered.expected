#disclaimer
  should: %s == "This whois service is provided by CentralNic Ltd and only contains information pertaining to Internet domain names we have registered for our customers. By using this service you are agreeing (1) not to use any information presented here for any purpose other than determining ownership of domain names, (2) not to store or reproduce this data in any way, (3) not to use any high-volume, automated, electronic processes to obtain data from this service. Abuse of this service is monitored and actions in contravention of these terms will result in being permanently blacklisted. All data is (c) CentralNic Ltd https://www.centralnic.com/"


#domain
  should: %s == "boerse.de.com"

#domain_id
  should: %s == "CNIC-DO819205"


#status
  should: %s == ["PENDING DELETE", "PENDING DELETE RESTORABLE"]

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2011-11-28 21:38:30 UTC")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-01-07 13:10:46 UTC")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-11-28 23:59:59 UTC")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == "H1167922"
  should: %s.name         == nil
  should: %s.organization == "Gandi SAS"
  should: %s.url          == "http://www.gandi.net/"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id            == "T2187-GANDI-VKXS"
  should: %s[0].name          == "Dhananjeyan Thangavelu"
  should: %s[0].organization  == "LTV Global Inc."
  should: %s[0].address       == "5 Meenakshi Nagar"
  should: %s[0].city          == "Trichy"
  should: %s[0].zip           == "620006"
  should: %s[0].state         == "TN"
  should: %s[0].country       == nil
  should: %s[0].country_code  == "IN"
  should: %s[0].phone         == "+91.4312431415"
  should: %s[0].fax           == nil
  should: %s[0].email         == "08cb0ba28467bf1e9ec03c83fdc62534-1376264@contact.gandi.net"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id            == "T2187-GANDI-VKXS"
  should: %s[0].name          == "Dhananjeyan Thangavelu"
  should: %s[0].organization  == "LTV Global Inc."
  should: %s[0].address       == "5 Meenakshi Nagar"
  should: %s[0].city          == "Trichy"
  should: %s[0].zip           == "620006"
  should: %s[0].state         == "TN"
  should: %s[0].country       == nil
  should: %s[0].country_code  == "IN"
  should: %s[0].phone         == "+91.4312431415"
  should: %s[0].fax           == nil
  should: %s[0].email         == "08cb0ba28467bf1e9ec03c83fdc62534-1376264@contact.gandi.net"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id            == "T2187-GANDI-VKXS"
  should: %s[0].name          == "Dhananjeyan Thangavelu"
  should: %s[0].organization  == "LTV Global Inc."
  should: %s[0].address       == "5 Meenakshi Nagar"
  should: %s[0].city          == "Trichy"
  should: %s[0].zip           == "620006"
  should: %s[0].state         == "TN"
  should: %s[0].country       == nil
  should: %s[0].country_code  == "IN"
  should: %s[0].phone         == "+91.4312431415"
  should: %s[0].fax           == nil
  should: %s[0].email         == "08cb0ba28467bf1e9ec03c83fdc62534-1376264@contact.gandi.net"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(3)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "b.dns.gandi.net"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "c.dns.gandi.net"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "a.dns.gandi.net"
